spamwatch/client.py

```diff
@@ -163,3 +163,13 @@ def delete_ban(self, user_id: int) -> None:
         """Remove a ban"""
         self._make_request(f'banlist/{user_id}', method='delete')
     # endregion
+    
+    #region Stats
+    def stats(self) -> Dict[str, int]:
+        """Get ban stats"""
+        try:
```

Comment:
You can remove the entire try except as the only error this route could cause would be a 500 which is handled in _make_request